client/components/DescriptionModule.js

```diff
@@ -44,54 +44,62 @@ const DescriptionModule = ({ gameid }) => {
 
   // On component mount, fetch relevant game information for component for a particular gameid (which is passed as props on mount)
   useEffect(() => {
-    fetchGamePhoto(gameid)
-      .then(res => {
-        if (res.error) {
-          throw new Error(res.error);
-        }
-        setGameThumbnail(res.url);
-      })
-      .catch(e => {
-        console.error(e);
-        // On fetchGamePhoto promise rejection, set default game thumbnail
-        setGameThumbnail('https://steamcdn-a.akamaihd.net/steam/apps/289070/header.jpg');
-      });
+    // fetchGamePhoto(gameid)
+    //   .then(res => {
+    //     if (res.error) {
+    //       throw new Error(res.error);
+    //     }
+    //     console.log(res.url);
+    //     setGameThumbnail(res.url);
+    //   })
+    //   .catch(e => {
+    //     console.error(e);
+    //     // On fetchGamePhoto promise rejection, set default game thumbnail
+    //     setGameThumbnail('https://steamcdn-a.akamaihd.net/steam/apps/289070/header.jpg');
+    //   });
+    setGameThumbnail('https://steamcdn-a.akamaihd.net/steam/apps/289070/header.jpg');
```

Comment:
Interesting: I knew the Photo Service was SUPPOSED to be providing this image - does this mean the description module had always been hard-coding the URL?

Either way, it might be worth trying to see if we can fix this for the final version. 

A request like:
http://localhost:3004/api/media/13

should return data like:
{"_id":"5f5b16af5027977f13af428d","id":13,"assets":[{"mediaType":"video","url":"https://www.youtube.com/embed/EneNc7DPrL8","thumbnail":"https://i.ytimg.com/vi/EneNc7DPrL8/default.jpg"},{"mediaType":"video","url":"https://www.youtube.com/embed/AWFaj4IQ4ro","thumbnail":"https://i.ytimg.com/vi/AWFaj4IQ4ro/default.jpg"},{"mediaType":"carouselPhoto","url":"https://res.cloudinary.com/dq3iywusm/image/upload/w_600,h_337/Sw_SuperMarioParty_090618_PressKit_SCRN_01_bmp_jpgcopy_ueow9o.jpg"},{"mediaType":"carouselPhoto","url":"https://res.cloudinary.com/dq3iywusm/image/upload/w_600,h_337/diab3reapcoll_cover_T4v3zN6-460x215_surhbh.jpg"},{"mediaType":"carouselPhoto","url":"https://res.cloudinary.com/dq3iywusm/image/upload/w_600,h_337/Screenshot181030.bmp_omq9xo.jpg"},{"mediaType":"carouselPhoto","url":"https://res.cloudinary.com/dq3iywusm/image/upload/w_600,h_337/PathOfExile_x64Steam_2016-11-21_21-23-31_itsdl1.jpg"},{"mediaType":"carouselPhoto","url":"https://res.cloudinary.com/dq3iywusm/image/upload/w_600,h_337/dota-2-game-screenshot-1_stjubh.jpg"},{"mediaType":"carouselPhoto","url":"https://res.cloudinary.com/dq3iywusm/image/upload/w_600,h_337/s8hg5_bbg1j2.jpg"}]}

Reply:
(To be clear, this is just an observation, not a critique)

Reply:
For UI to work, for right now, I set it as default thumbnail picture. Since our group is not doing photo service im trying to see where i should genrate the data locally and provide it